terraform/modules/aws-brig-email-sending/outputs.tf

```diff
@@ -0,0 +1,14 @@
+# Output required to configure wire-server
+
+output "ses_endpoint" {
+  value = "https://email.${var.region}.amazonaws.com"
+}
+
+output "brig_access_key" {
+  value = aws_iam_access_key.brig.id
+}
+
+# This value is sensitive in nature and you cannot get it from AWS later.
```

Comment:
Please apply the same refactoring on this PR than on the other PR about this comment.

Reply:
I guess you are referring to [this](https://github.com/wireapp/wire-server-deploy/pull/243/files#diff-37fe5bb4975cf90f7786a6ad418fab3dR15-R17) change. Since you didnt reply [here](https://github.com/wireapp/wire-server-deploy/pull/243#discussion_r416514296), I guess, this was what you had in mind?

Reply:
never mind. saw [your comment](https://github.com/wireapp/wire-server-deploy/pull/244#discussion_r416607447)